pkg/module/parse/template_to_module.go

```diff
@@ -32,26 +33,31 @@ type Parser struct {
 	InKCPMode           bool
 	remoteSyncNamespace string
 	*ocmextensions.ComponentDescriptorCache
+	EnableVerification bool
+	PublicKeyFilePath  string
 }
 
 func NewParser(
 	clnt client.Client,
 	descriptorCache *ocmextensions.ComponentDescriptorCache,
 	inKCPMode bool,
 	remoteSyncNamespace string,
+	enableVerification bool,
+	publicKeyFilePath string,
 ) *Parser {
 	return &Parser{
 		Client:                   clnt,
 		ComponentDescriptorCache: descriptorCache,
 		InKCPMode:                inKCPMode,
 		remoteSyncNamespace:      remoteSyncNamespace,
+		EnableVerification:       enableVerification,
+		PublicKeyFilePath:        publicKeyFilePath,
 	}
 }
 
-func (p *Parser) GenerateModulesFromTemplates(ctx context.Context,
+func (parser *Parser) GenerateModulesFromTemplates(ctx context.Context,
 	kyma *v1beta2.Kyma,
```

Comment:
I prefer to keep it as p instead of parser.

Reply:
To me using full name is always better for readability purposes, I actually haven't found strict rule defined by idiomatic Go for the naming of the receiver, and by this article https://blog.heroku.com/neither-self-nor-this-receivers-in-go#naming-the-receiver, actually there is no restriction for this, if you find use full name is an antipattern, I'm free to change it, otherwise I still prefer use fullname, don't like this short name, especially for long function. 
```
If the name of the struct is Server, we'll usually see s or srv or even server. 
All of these are fine—short is convenient, but it's more about uniquely identifying the variable in a consistent way.
```

Reply:
I got your point, just for the purpose of the consistency with other receivers I prefer it with `p` instead of `parser`.
Of course, I strongly disagree of use of `this` or `self` for example as its described on the previous link as it;s remind to oop language.
Please if you don't mind change it back to `p`!